api/src/main/java/org/pmiops/workbench/interceptors/AuthInterceptor.java

```diff
@@ -155,18 +155,7 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
       // TODO(danrodney): start populating contact email in Google account, use it here.
       user =
           userService.createUser(
-              userInfo.getGivenName(),
-              userInfo.getFamilyName(),
-              userInfo.getEmail(),
```

Comment:
`userInfo` is a Google OAuth2 `Userinfoplus` object so "email" is correct from that perspective, though it's a "userName" to Workbench.

Reply:
Can you put that comment in the code and replace `userInfo` with something like `googleOAuth2id`?

Reply:
done